azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/storage/StorageAutoConfiguration.java

```diff
@@ -53,7 +54,7 @@ public ServiceURL createServiceUrl(@Autowired(required = false) PipelineOptions
 
         final SharedKeyCredentials credentials = new SharedKeyCredentials(properties.getAccountName(),
                 properties.getAccountKey());
-        final URL blobUrl = new URL(String.format(BLOB_URL, properties.getAccountName()));
+        final URL blobUrl = properties.isEnableHttps() ? new URL(String.format(BLOB_Https_URL, properties.getAccountName())) : new URL(String.format(BLOB_URL, properties.getAccountName()));
```

Comment:
This line is too long, please restrict the character length within 120 characters.